<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<style>body{background-color:white;}</style>


</head>
<body>
<div id="nxbhjvpvcj" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
  <style>#nxbhjvpvcj table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#nxbhjvpvcj thead, #nxbhjvpvcj tbody, #nxbhjvpvcj tfoot, #nxbhjvpvcj tr, #nxbhjvpvcj td, #nxbhjvpvcj th {
  border-style: none;
}

#nxbhjvpvcj p {
  margin: 0;
  padding: 0;
}

#nxbhjvpvcj .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#nxbhjvpvcj .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#nxbhjvpvcj .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#nxbhjvpvcj .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#nxbhjvpvcj .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#nxbhjvpvcj .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#nxbhjvpvcj .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#nxbhjvpvcj .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#nxbhjvpvcj .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#nxbhjvpvcj .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#nxbhjvpvcj .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#nxbhjvpvcj .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#nxbhjvpvcj .gt_spanner_row {
  border-bottom-style: hidden;
}

#nxbhjvpvcj .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#nxbhjvpvcj .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#nxbhjvpvcj .gt_from_md > :first-child {
  margin-top: 0;
}

#nxbhjvpvcj .gt_from_md > :last-child {
  margin-bottom: 0;
}

#nxbhjvpvcj .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#nxbhjvpvcj .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#nxbhjvpvcj .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#nxbhjvpvcj .gt_row_group_first td {
  border-top-width: 2px;
}

#nxbhjvpvcj .gt_row_group_first th {
  border-top-width: 2px;
}

#nxbhjvpvcj .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#nxbhjvpvcj .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#nxbhjvpvcj .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#nxbhjvpvcj .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#nxbhjvpvcj .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#nxbhjvpvcj .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#nxbhjvpvcj .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#nxbhjvpvcj .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#nxbhjvpvcj .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#nxbhjvpvcj .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#nxbhjvpvcj .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#nxbhjvpvcj .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#nxbhjvpvcj .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#nxbhjvpvcj .gt_left {
  text-align: left;
}

#nxbhjvpvcj .gt_center {
  text-align: center;
}

#nxbhjvpvcj .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#nxbhjvpvcj .gt_font_normal {
  font-weight: normal;
}

#nxbhjvpvcj .gt_font_bold {
  font-weight: bold;
}

#nxbhjvpvcj .gt_font_italic {
  font-style: italic;
}

#nxbhjvpvcj .gt_super {
  font-size: 65%;
}

#nxbhjvpvcj .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#nxbhjvpvcj .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#nxbhjvpvcj .gt_indent_1 {
  text-indent: 5px;
}

#nxbhjvpvcj .gt_indent_2 {
  text-indent: 10px;
}

#nxbhjvpvcj .gt_indent_3 {
  text-indent: 15px;
}

#nxbhjvpvcj .gt_indent_4 {
  text-indent: 20px;
}

#nxbhjvpvcj .gt_indent_5 {
  text-indent: 25px;
}
</style>
  <table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    <tr class="gt_heading">
      <td colspan="28" class="gt_heading gt_title gt_font_normal" style><strong>Renal Cell Carcinoma</strong></td>
    </tr>
    <tr class="gt_heading">
      <td colspan="28" class="gt_heading gt_subtitle gt_font_normal gt_bottom_border" style>TF Motifs</td>
    </tr>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="seqnames">seqnames</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="start">start</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="end">end</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="width">width</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="strand">strand</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="SNP_id">SNP_id</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="REF">REF</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="ALT">ALT</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="varType">varType</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="motifPos">motifPos</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="geneSymbol">geneSymbol</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="dataSource">dataSource</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="providerName">providerName</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="providerId">providerId</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="seqMatch">seqMatch</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="pctRef">pctRef</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="pctAlt">pctAlt</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="scoreRef">scoreRef</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="scoreAlt">scoreAlt</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="Refpvalue">Refpvalue</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="Altpvalue">Altpvalue</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="snpPos">snpPos</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="alleleRef">alleleRef</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="alleleAlt">alleleAlt</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="effect">effect</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="altPos">altPos</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="alleleDiff">alleleDiff</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" style="background-color: #BEBEBE; color: #FFFFFF; font-weight: bold;" scope="col" id="alleleEffectSize">alleleEffectSize</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804709</td>
<td headers="end" class="gt_row gt_right">124804709</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">+</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466948</td>
<td headers="REF" class="gt_row gt_left">G</td>
<td headers="ALT" class="gt_row gt_left">A</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-13, -1</td>
<td headers="geneSymbol" class="gt_row gt_left">NA</td>
<td headers="dataSource" class="gt_row gt_left">stamlab</td>
<td headers="providerName" class="gt_row gt_left">UW.Motif.0497</td>
<td headers="providerId" class="gt_row gt_left">UW.Motif.0497</td>
<td headers="seqMatch" class="gt_row gt_left">        attatacttcactGtcttcagggtgaa          </td>
<td headers="pctRef" class="gt_row gt_right">0.8744</td>
<td headers="pctAlt" class="gt_row gt_right">0.7490</td>
<td headers="scoreRef" class="gt_row gt_right">5.1607</td>
<td headers="scoreAlt" class="gt_row gt_right">4.4756</td>
<td headers="Refpvalue" class="gt_row gt_right">1.028888e-04</td>
<td headers="Altpvalue" class="gt_row gt_right">1.942281e-03</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">weak</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">−0.6851</td>
<td headers="alleleEffectSize" class="gt_row gt_right">−0.1172</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">-</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">C</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-5, 6</td>
<td headers="geneSymbol" class="gt_row gt_left">E2F1</td>
<td headers="dataSource" class="gt_row gt_left">jolma2013</td>
<td headers="providerName" class="gt_row gt_left">Hsapiens-jolma2013-E2F1-2</td>
<td headers="providerId" class="gt_row gt_left">E2F1</td>
<td headers="seqMatch" class="gt_row gt_left">          ttagagctgggCgccagcttaat            </td>
<td headers="pctRef" class="gt_row gt_right">0.7193</td>
<td headers="pctAlt" class="gt_row gt_right">0.8578</td>
<td headers="scoreRef" class="gt_row gt_right">5.3125</td>
<td headers="scoreAlt" class="gt_row gt_right">6.3011</td>
<td headers="Refpvalue" class="gt_row gt_right">2.435684e-03</td>
<td headers="Altpvalue" class="gt_row gt_right">8.392334e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">strong</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.9886</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.1351</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">-</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">C</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-5, 6</td>
<td headers="geneSymbol" class="gt_row gt_left">E2F1</td>
<td headers="dataSource" class="gt_row gt_left">jolma2013</td>
<td headers="providerName" class="gt_row gt_left">Hsapiens-jolma2013-E2F1</td>
<td headers="providerId" class="gt_row gt_left">E2F1</td>
<td headers="seqMatch" class="gt_row gt_left">          ttagagctgggCgccagcttaat            </td>
<td headers="pctRef" class="gt_row gt_right">0.7479</td>
<td headers="pctAlt" class="gt_row gt_right">0.8806</td>
<td headers="scoreRef" class="gt_row gt_right">5.7228</td>
<td headers="scoreAlt" class="gt_row gt_right">6.7131</td>
<td headers="Refpvalue" class="gt_row gt_right">1.358211e-03</td>
<td headers="Altpvalue" class="gt_row gt_right">5.048513e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">strong</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.9903</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.1302</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">-</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">A</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-7, 13</td>
<td headers="geneSymbol" class="gt_row gt_left">ZFP42</td>
<td headers="dataSource" class="gt_row gt_left">jaspar2022</td>
<td headers="providerName" class="gt_row gt_left">MA1651.1</td>
<td headers="providerId" class="gt_row gt_left">MA1651.1</td>
<td headers="seqMatch" class="gt_row gt_left"> gcatgaaggttagagctgggAgccagcttaatctctttcat   </td>
<td headers="pctRef" class="gt_row gt_right">0.7473</td>
<td headers="pctAlt" class="gt_row gt_right">0.8056</td>
<td headers="scoreRef" class="gt_row gt_right">6.7229</td>
<td headers="scoreAlt" class="gt_row gt_right">7.2128</td>
<td headers="Refpvalue" class="gt_row gt_right">4.185419e-04</td>
<td headers="Altpvalue" class="gt_row gt_right">8.517892e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">weak</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.4899</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.0554</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">+</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">C</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-7, 9</td>
<td headers="geneSymbol" class="gt_row gt_left">E2F3</td>
<td headers="dataSource" class="gt_row gt_left">SwissRegulon</td>
<td headers="providerName" class="gt_row gt_left">E2F3.SwissRegulon</td>
<td headers="providerId" class="gt_row gt_left">E2F3.SwissRegulon</td>
<td headers="seqMatch" class="gt_row gt_left">     gaaggttagagctgggCgccagcttaatctctt       </td>
<td headers="pctRef" class="gt_row gt_right">0.7671</td>
<td headers="pctAlt" class="gt_row gt_right">0.9173</td>
<td headers="scoreRef" class="gt_row gt_right">4.9886</td>
<td headers="scoreAlt" class="gt_row gt_right">5.9063</td>
<td headers="Refpvalue" class="gt_row gt_right">1.483318e-03</td>
<td headers="Altpvalue" class="gt_row gt_right">4.261162e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">strong</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.9177</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.1431</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">-</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">C</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-5, 6</td>
<td headers="geneSymbol" class="gt_row gt_left">E2F1</td>
<td headers="dataSource" class="gt_row gt_left">jaspar2022</td>
<td headers="providerName" class="gt_row gt_left">MA0024.3</td>
<td headers="providerId" class="gt_row gt_left">MA0024.3</td>
<td headers="seqMatch" class="gt_row gt_left">          ttagagctgggCgccagcttaat            </td>
<td headers="pctRef" class="gt_row gt_right">0.7193</td>
<td headers="pctAlt" class="gt_row gt_right">0.8578</td>
<td headers="scoreRef" class="gt_row gt_right">5.3139</td>
<td headers="scoreAlt" class="gt_row gt_right">6.3029</td>
<td headers="Refpvalue" class="gt_row gt_right">2.435684e-03</td>
<td headers="Altpvalue" class="gt_row gt_right">8.392334e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">strong</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.9889</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.1351</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">-</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">A</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">1, 15</td>
<td headers="geneSymbol" class="gt_row gt_left">NA</td>
<td headers="dataSource" class="gt_row gt_left">stamlab</td>
<td headers="providerName" class="gt_row gt_left">UW.Motif.0384</td>
<td headers="providerId" class="gt_row gt_left">UW.Motif.0384</td>
<td headers="seqMatch" class="gt_row gt_left">      aaggttagagctgggAgccagcttaatctct        </td>
<td headers="pctRef" class="gt_row gt_right">0.7529</td>
<td headers="pctAlt" class="gt_row gt_right">0.8825</td>
<td headers="scoreRef" class="gt_row gt_right">4.3891</td>
<td headers="scoreAlt" class="gt_row gt_right">5.0855</td>
<td headers="Refpvalue" class="gt_row gt_right">2.147075e-03</td>
<td headers="Altpvalue" class="gt_row gt_right">7.318053e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">weak</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.6964</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.1218</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804709</td>
<td headers="end" class="gt_row gt_right">124804709</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">-</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466948</td>
<td headers="REF" class="gt_row gt_left">G</td>
<td headers="ALT" class="gt_row gt_left">A</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-15, -1</td>
<td headers="geneSymbol" class="gt_row gt_left">NA</td>
<td headers="dataSource" class="gt_row gt_left">stamlab</td>
<td headers="providerName" class="gt_row gt_left">UW.Motif.0337</td>
<td headers="providerId" class="gt_row gt_left">UW.Motif.0337</td>
<td headers="seqMatch" class="gt_row gt_left">      ccattatacttcactGtcttcagggtgaagg        </td>
<td headers="pctRef" class="gt_row gt_right">0.8812</td>
<td headers="pctAlt" class="gt_row gt_right">0.7611</td>
<td headers="scoreRef" class="gt_row gt_right">5.1831</td>
<td headers="scoreAlt" class="gt_row gt_right">4.5293</td>
<td headers="Refpvalue" class="gt_row gt_right">6.516511e-05</td>
<td headers="Altpvalue" class="gt_row gt_right">1.061688e-03</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">weak</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">−0.6539</td>
<td headers="alleleEffectSize" class="gt_row gt_right">−0.1122</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">-</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">A</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-4, 10</td>
<td headers="geneSymbol" class="gt_row gt_left">NA</td>
<td headers="dataSource" class="gt_row gt_left">stamlab</td>
<td headers="providerName" class="gt_row gt_left">UW.Motif.0132</td>
<td headers="providerId" class="gt_row gt_left">UW.Motif.0132</td>
<td headers="seqMatch" class="gt_row gt_left">       aggttagagctgggAgccagcttaatctc         </td>
<td headers="pctRef" class="gt_row gt_right">0.7592</td>
<td headers="pctAlt" class="gt_row gt_right">0.8888</td>
<td headers="scoreRef" class="gt_row gt_right">4.4494</td>
<td headers="scoreAlt" class="gt_row gt_right">5.1478</td>
<td headers="Refpvalue" class="gt_row gt_right">1.506497e-03</td>
<td headers="Altpvalue" class="gt_row gt_right">4.696101e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">weak</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.6984</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.1215</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">-</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">A</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-6, 2</td>
<td headers="geneSymbol" class="gt_row gt_left">NA</td>
<td headers="dataSource" class="gt_row gt_left">stamlab</td>
<td headers="providerName" class="gt_row gt_left">UW.Motif.0022</td>
<td headers="providerId" class="gt_row gt_left">UW.Motif.0022</td>
<td headers="seqMatch" class="gt_row gt_left">             gagctgggAgccagctt               </td>
<td headers="pctRef" class="gt_row gt_right">0.7958</td>
<td headers="pctAlt" class="gt_row gt_right">0.9014</td>
<td headers="scoreRef" class="gt_row gt_right">4.5467</td>
<td headers="scoreAlt" class="gt_row gt_right">5.1183</td>
<td headers="Refpvalue" class="gt_row gt_right">5.073547e-04</td>
<td headers="Altpvalue" class="gt_row gt_right">4.959106e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">weak</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.5717</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.1011</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">+</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">C</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-6, 7</td>
<td headers="geneSymbol" class="gt_row gt_left">E2F1</td>
<td headers="dataSource" class="gt_row gt_left">jolma2013</td>
<td headers="providerName" class="gt_row gt_left">Hsapiens-jolma2013-E2F1-3</td>
<td headers="providerId" class="gt_row gt_left">E2F1</td>
<td headers="seqMatch" class="gt_row gt_left">        ggttagagctgggCgccagcttaatct          </td>
<td headers="pctRef" class="gt_row gt_right">0.7069</td>
<td headers="pctAlt" class="gt_row gt_right">0.8349</td>
<td headers="scoreRef" class="gt_row gt_right">5.4313</td>
<td headers="scoreAlt" class="gt_row gt_right">6.3838</td>
<td headers="Refpvalue" class="gt_row gt_right">2.447356e-03</td>
<td headers="Altpvalue" class="gt_row gt_right">8.805841e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">strong</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.9525</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.1251</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">+</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">C</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-9, 5</td>
<td headers="geneSymbol" class="gt_row gt_left">ZNF417</td>
<td headers="dataSource" class="gt_row gt_left">jaspar2022</td>
<td headers="providerName" class="gt_row gt_left">MA1727.1</td>
<td headers="providerId" class="gt_row gt_left">MA1727.1</td>
<td headers="seqMatch" class="gt_row gt_left">       aggttagagctgggCgccagcttaatctc         </td>
<td headers="pctRef" class="gt_row gt_right">0.8607</td>
<td headers="pctAlt" class="gt_row gt_right">0.9724</td>
<td headers="scoreRef" class="gt_row gt_right">6.6043</td>
<td headers="scoreAlt" class="gt_row gt_right">7.4280</td>
<td headers="Refpvalue" class="gt_row gt_right">7.788464e-05</td>
<td headers="Altpvalue" class="gt_row gt_right">5.127862e-06</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">strong</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.8237</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.1079</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804709</td>
<td headers="end" class="gt_row gt_right">124804709</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">-</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466948</td>
<td headers="REF" class="gt_row gt_left">G</td>
<td headers="ALT" class="gt_row gt_left">A</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-5, 7</td>
<td headers="geneSymbol" class="gt_row gt_left">NA</td>
<td headers="dataSource" class="gt_row gt_left">stamlab</td>
<td headers="providerName" class="gt_row gt_left">UW.Motif.0228</td>
<td headers="providerId" class="gt_row gt_left">UW.Motif.0228</td>
<td headers="seqMatch" class="gt_row gt_left">         ttatacttcactGtcttcagggtga           </td>
<td headers="pctRef" class="gt_row gt_right">0.8778</td>
<td headers="pctAlt" class="gt_row gt_right">0.7535</td>
<td headers="scoreRef" class="gt_row gt_right">5.1713</td>
<td headers="scoreAlt" class="gt_row gt_right">4.4882</td>
<td headers="Refpvalue" class="gt_row gt_right">7.013977e-05</td>
<td headers="Altpvalue" class="gt_row gt_right">1.516566e-03</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">weak</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">−0.6831</td>
<td headers="alleleEffectSize" class="gt_row gt_right">−0.1169</td></tr>
    <tr><td headers="seqnames" class="gt_row gt_center">chr7</td>
<td headers="start" class="gt_row gt_right">124804846</td>
<td headers="end" class="gt_row gt_right">124804846</td>
<td headers="width" class="gt_row gt_right">1</td>
<td headers="strand" class="gt_row gt_center">+</td>
<td headers="SNP_id" class="gt_row gt_left">rs6466949</td>
<td headers="REF" class="gt_row gt_left">T</td>
<td headers="ALT" class="gt_row gt_left">A</td>
<td headers="varType" class="gt_row gt_left">SNV</td>
<td headers="motifPos" class="gt_row gt_center">-7, 15</td>
<td headers="geneSymbol" class="gt_row gt_left">ZNF16</td>
<td headers="dataSource" class="gt_row gt_left">jaspar2022</td>
<td headers="providerName" class="gt_row gt_left">MA1654.1</td>
<td headers="providerId" class="gt_row gt_left">MA1654.1</td>
<td headers="seqMatch" class="gt_row gt_left">ggcatgaaggttagagctgggAgccagcttaatctctttcattt </td>
<td headers="pctRef" class="gt_row gt_right">0.6203</td>
<td headers="pctAlt" class="gt_row gt_right">0.6921</td>
<td headers="scoreRef" class="gt_row gt_right">8.9482</td>
<td headers="scoreAlt" class="gt_row gt_right">9.9434</td>
<td headers="Refpvalue" class="gt_row gt_right">9.283294e-04</td>
<td headers="Altpvalue" class="gt_row gt_right">8.551161e-05</td>
<td headers="snpPos" class="gt_row gt_right">NA</td>
<td headers="alleleRef" class="gt_row gt_right">NA</td>
<td headers="alleleAlt" class="gt_row gt_right">NA</td>
<td headers="effect" class="gt_row gt_left">strong</td>
<td headers="altPos" class="gt_row gt_right">1</td>
<td headers="alleleDiff" class="gt_row gt_right">0.9952</td>
<td headers="alleleEffectSize" class="gt_row gt_right">0.0700</td></tr>
  </tbody>
  
  
</table>
</div>
</body>
</html>
